### PR TITLE
Replaced d2l-audio and d2l-video with ui-labs media-player

### PR DESCRIPTION
**Pull request by ktonon**
Added demo pages for video:

Screen Shot 2020-10-26 at 11 35 28 AM

and audio:

Screen Shot 2020-10-26 at 11 35 38 AM

is there a way that I can test this in a dev LMS?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,9 +65,8 @@
   "main": "d2l-sequences.js",
   "dependencies": {
     "@brightspace-ui-labs/accordion": "^2.3.0",
+    "@brightspace-ui-labs/media-player": "^1.8.0",
     "@brightspace-ui/core": "^1",
-    "@d2l/audio": "Brightspace/d2l-audio#semver:^3",
```
Comment (dbatiste) [inline package.json:69]: 🎉

**Comment by ktonon** [comment]
I don't know how to resolve the sauce creds issue in ci

**Comment by sarahboyd** [comment]
> I don't know how to resolve the sauce creds issue in ci

it's because you're using a fork, if you use a branch instead sauce should work

**Comment by ktonon** [comment]
> > I don't know how to resolve the sauce creds issue in ci
> 
> it's because you're using a fork, if you use a branch instead sauce should work

Thanks. See #274

**Pull request closed without merging**